src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -886,4 +848,42 @@ final case class TenantApiServiceImpl(
       getTenantByExternalIdResponse[Tenant](operationLabel)(getTenantByExternalId200)
     }
   }
+
+  override def addTenantMail(tenantId: String, mailSeed: MailSeed)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE, M2M_ROLE, INTERNAL_ROLE) {
```

Comment:
@beetlecrunch are there any restrictions on roles who may add the mail?

Reply:
```suggestion
  ): Route = authorize(ADMIN_ROLE, API_ROLE) {
```